From: esote <25505632+esote@users.noreply.github.com>
Subject: [PATCH] Remove rc NFS reference

Notes: Apply to /etc/rc.

---
 rc | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/rc b/rc
--- a/rc
+++ b/rc
@@ -414,14 +414,6 @@ if [[ $pf != NO ]]; then
 	pass in proto carp keep state (no-sync)
 	pass out proto carp !received-on any keep state (no-sync)"
 
-	if (($(sysctl -n vfs.mounts.nfs 2>/dev/null) > 0)); then
-		# Don't kill NFS.
-		RULES="set reassemble yes no-df
-		$RULES
-		pass in proto { tcp, udp } from any port { sunrpc, nfsd } to any
-		pass out proto { tcp, udp } from any to any port { sunrpc, nfsd } !received-on any"
-	fi
-
 	print -- "$RULES" | pfctl -f -
 	pfctl -e
 fi
-- 
2.20.1
